project/Dependencies.scala

```diff
@@ -4,8 +4,10 @@ object Dependencies {
 
   def playJson(scalaVersion: String): Seq[ModuleID] = {
     CrossVersion.partialVersion(scalaVersion) match {
-      case Some((2, 11)) => Seq("com.typesafe.play" %% "play-json" % "2.5.19")
-      case _ => Seq("com.typesafe.play" %% "play-json" % "2.7.4", "com.typesafe.play" %% "play-json-joda" % "2.7.4")
+      case Some((2, 11)) => Seq("com.typesafe.play" %% "play-json" % sys.props.getOrElse("playVersion", "2.5.19"))
```

Comment:
you can be less lazy and get `playVersion` on the top, still... doesn't matter that much

Reply:
I did not do it because I need different defaults. But could move the options up